docs/go-logging-instrumentation.md

```diff
@@ -0,0 +1,113 @@
+---
+id: go-logging-instrumentation
+title: Scope Go Agent Logging Instrumentation
+sidebar_label: Logging Instrumentation
+---
+
+The Scope Go agent has support for capturing logs from the following sources:
+
+- Test logs (calls to `t.Log()` and other test methods) are automatically instrumented using monkey patching. Includes support for capturing source file and number information, and also for the exact timestamp and level of the log event. It is compatible with parallel tests.
+- Scope Test logs (calls to `test.Log` and other test methods) includes support for capturing source file and number information, and also for the exact timestamp and level of the log event. It is compatible with parallel tests.
+- Standard logger (calls to `log.Println()` and related methods) is automatically instrumented. Includes support for capturing source file and number information, and also for the exact timestamp of the log event. It is not compatible with parallel tests - if logs are emitted through the standard logger while tests are running in parallel (using `t.Parallel()`), logs are attached to any running test at that time.
+- Custom loggers (`log.Logger` instances) can be manually instrumented with the same capabilities as the standard logger autoinstrumentation.
+- Standard output/error (`os.Stdout` and `os.Stderr`) can be manually instrumented. Includes support for the timestamp of the log event but not the source file or line number information. It is not compatible with parallel tests.
+
+> Note that instrumentation for `log.Logger` and standard output/error is done globally in the application, and is not compatible with parallel tests. In order to differentiate the logs of a given test, only sequential tests are supported for these instrumentations. Only per-test logs (calls to `t.Log()` and other test methods) support parallel tests.
+
+> The Test logs instrumentation can be disabled adding the environment variable: **`SCOPE_DISABLE_MONKEY_PATCHING=true`**
+
+## Scope Test logs
+
+In case the standard Test logs instrumentation is disabled, the Scope Test struct has support for logging and is completly instrumented. As the Test logs instrumentation, support all features (source file and number, exact timestamp, and level of the log).
```

Comment:
```suggestion
In case the standard library test automatic logging instrumentation is disabled (by using the environment variable `SCOPE_DISABLE_MONKEY_PATCHING=true`), the Scope `test` struct implements the same logging interface as `logging.T` and it's instrumented. It also support all features, such as capturing the source file and line number, exact timestamp, and level of the log.
```